packages/manager/modules/billing-components/src/components/cancellation-form/billing-confirmTerminate.controller.js

```diff
@@ -38,4 +37,32 @@ export default class TerminateServiceCtrl {
         this.loading = false;
       });
   }
+
+  terminateByoipService() {
+    return this.BillingTerminate.terminateByoipService(this.serviceId)
+      .then(() =>
```

Comment:
then, catch and finally block seems duplicate of terminateService call. Can we avoid duplication ?

Reply:
@sachinrameshn  moved to common function. Thanks.